tests/llm_translation/test_bedrock_completion.py

```diff
@@ -2535,6 +2535,7 @@ def test_bedrock_error_handling_streaming():
 )
 @pytest.mark.flaky(retries=6, delay=2)
 @pytest.mark.asyncio
+@pytest.mark.skip(reason="Bedrock Raising Internal Server Error for this API call")
```

Comment:
note - this was bedrock API that was having an outage for this specific request. 

```
BedrockException - {"message":"The system encountered an unexpected error during processing. Try your request again."}
```